jenkins/README.md

```diff
@@ -0,0 +1,24 @@
+# How to work with Jenkins jobs
+
+All the job scripts are in the [test-infra/jobs/{job-name}.sh](https://github.com/kubernetes/test-infra/tree/master/jobs) files now.
```

Comment:
May want to note that we plan on migrating these into yaml files in the near future.